src/theme/chat/_message.scss

```diff
@@ -27,9 +27,10 @@
 		padding: 8px 16px 16px 16px;
 	}
 	.mentioned_fa6fd2 {
-		background: rgb(var(--chat-mention-colour), 0.05);
+		$color: 'rgb(from var(--chat-mention-colour) r g b, 0.05)';
```

Comment:
That's unfortunately needed because in the bd-sass package, sass is too old and this wasn't merged yet: https://github.com/sass/sass/issues/3673
I've tried to override it via pnpm but it didn't work

Reply:
I've updated the `bd-scss` package which now allows this. Sorry about that.

You still need to wrap the `src/core/_root.scss` -> `--accent` variable with `rgb`

Reply:
I just fixed all the issues in the latest commits!